.github/workflows/validate-python.yaml

```diff
@@ -45,6 +50,6 @@ jobs:
       working-directory: ${{ github.workspace }}/orchestration/dagster_orchestration
       if: always() && steps.dependencies.outcome == 'success' # run all three checks, even if a prior check fails
     - name: Run test suite
-      run: poetry run pytest
+      run: poetry run pytest -v
```

Comment:
I'm having some trouble finding what this flag/arg does; anyone care to enlighten me? `-v` usually means version to me which I'm guessing we're not doing here, and my 5 minutes of googling has not turned up anything that makes sense

Reply:
this is the 'verbose' flag, it controls how much gets output to the console

Reply:
Yeah I added this because I wanted a bit of feedback during the test cycle, happy to remove if folks feel otherwise.